<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayWindow</class>
 <widget class="QMainWindow" name="DisplayWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DisplayWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: white;</string>
  </property>
  <widget class="QWidget" name="central_widget">
   <layout class="QVBoxLayout" name="main_layout" stretch="3,0,12,0,2">
    <item>
     <layout class="QGridLayout" name="header_layout" columnstretch="3,6,0,2">
      <property name="horizontalSpacing">
       <number>36</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="time_label">
        <property name="styleSheet">
         <string notr="true">font-size: 20pt;
font-weight: bold;
font-family: Fira Mono;
</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="sentence_text_label">
        <property name="styleSheet">
         <string notr="true">font-size: 14pt;
font-family: Microsoft Yahei;
</string>
        </property>
        <property name="text">
         <string>不在沉默中爆发，就在沉默中灭亡。</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="sentence_author_label">
        <property name="styleSheet">
         <string notr="true">font-size: 14pt;
font-family: Microsoft Yahei;
</string>
        </property>
        <property name="text">
         <string>——鲁迅</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="date_weekday_label">
        <property name="styleSheet">
         <string notr="true">font-size: 14pt;
font-family: Microsoft Yahei;
</string>
        </property>
        <property name="text">
         <string>1月1日 星期一</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignHCenter|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="TitleWidget" name="title_widget" native="true"/>
      </item>
      <item row="0" column="2" rowspan="2">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="header_body_line">
      <property name="styleSheet">
       <string notr="true">color: rgba(164, 172, 187, 80);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="body_layout" rowstretch="0,1" columnstretch="2,0,3,0,2">
      <item row="0" column="4">
       <widget class="QLabel" name="lessons_title_label">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt;
font-weight: bold;
font-family: Microsoft YaHei;
</string>
        </property>
        <property name="text">
         <string>课程</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="notices_title_label">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt;
font-weight: bold;
font-family: Microsoft YaHei;
</string>
        </property>
        <property name="text">
         <string>公告</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="arrangement_title_label">
        <property name="styleSheet">
         <string notr="true">font-size: 18pt;
font-weight: bold;
font-family: Microsoft YaHei;
</string>
        </property>
        <property name="text">
         <string>安排</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="notices_layout" stretch="0,0,0,1">
        <item>
         <widget class="QLabel" name="notices_data_title_label">
          <property name="styleSheet">
           <string notr="true">font-size: 16pt;
font-family: Microsoft YaHei;
</string>
          </property>
          <property name="text">
           <string>假期通知</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="notices_date_label">
          <property name="styleSheet">
           <string notr="true">font-size: 15pt;
font-family: Microsoft YaHei;
font-weight: 300;</string>
          </property>
          <property name="text">
           <string>至3月1日</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QTextBrowser" name="notices_text_browser">
          <property name="styleSheet">
           <string notr="true">font-size: 18pt;
font-family: Microsoft YaHei;

background: transparent;</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="html">
           <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Microsoft YaHei'; font-size:18pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好好&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1" rowspan="2">
       <widget class="Line" name="notices_arrangement_line">
        <property name="styleSheet">
         <string notr="true">color: rgba(164, 172, 187, 80);</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="3" rowspan="2">
       <widget class="Line" name="arrangement_lessons_line">
        <property name="styleSheet">
         <string notr="true">color: rgba(164, 172, 187, 80);</string>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QWidget" name="arrangement_widget" native="true"/>
      </item>
      <item row="1" column="4">
       <widget class="QWidget" name="lessons_widget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="body_footer_line">
      <property name="styleSheet">
       <string notr="true">color: rgba(164, 172, 187, 80);</string>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth">
       <number>2</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="weather_layout">
      <property name="spacing">
       <number>4</number>
      </property>
      <item>
       <widget class="WeatherWidget" name="weather_widget_1" native="true">
        <property name="styleSheet">
         <string notr="true">color: #ee8545;
font-weight: bold;</string>
        </property>
        <property name="class" stdset="0">
         <string>weather-today</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="WeatherWidget" name="weather_widget_2" native="true"/>
      </item>
      <item>
       <widget class="WeatherWidget" name="weather_widget_3" native="true"/>
      </item>
      <item>
       <widget class="WeatherWidget" name="weather_widget_4" native="true"/>
      </item>
      <item>
       <widget class="WeatherWidget" name="weather_widget_5" native="true"/>
      </item>
      <item>
       <widget class="WeatherWidget" name="weather_widget_6" native="true"/>
      </item>
      <item>
       <widget class="WeatherWidget" name="weather_widget_7" native="true"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TitleWidget</class>
   <extends>QWidget</extends>
   <header>titlewidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WeatherWidget</class>
   <extends>QWidget</extends>
   <header>weatherwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
